src/psycopmlutils/timeseriesflattener/data_integrity.py

```diff
@@ -0,0 +1,241 @@
+"""Code to generate data integrity and train/val/test drift reports."""
+from pathlib import Path
+from typing import List, Optional
+
+from deepchecks.core.suite import SuiteResult
+from deepchecks.tabular import Dataset, Suite
+from deepchecks.tabular.checks import (
+    CategoryMismatchTrainTest,
+    DatasetsSizeComparison,
+    FeatureLabelCorrelation,
+    FeatureLabelCorrelationChange,
+    IdentifierLabelCorrelation,
+    IndexTrainTestLeakage,
+    NewLabelTrainTest,
+    TrainTestLabelDrift,
+)
+from deepchecks.tabular.suites import data_integrity
+from wasabi import Printer
+
+from psycopmlutils.loaders.flattened import load_split_outcomes, load_split_predictors
+
+
+def check_feature_sets_dir(
```

Comment:
```suggestion
def check_feature_set_integrity_from_dir(
```

Think this is a bit more clear, hope you don't find it too nitpicky ;-)

OBS: Recommend renaming with F2, rather than accepting this suggestion.